apps/vr-tests-react-components/src/stories/CalendarCompat.stories.tsx

```diff
@@ -1,38 +1,51 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31915&commitId=a0e79906f420a5676054a4e49b2ad3e89478df27&env=prod&clientType=FLUENTUI)


  <details><summary><b>Calendar Compat</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar Compat.Custom icons - RTL.default.chromium.png|0| Added|
|Calendar Compat.Default - RTL.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>